File: app/src/main/java/com/porkandlager/travelonbudget/mvp/FlightDetail/FlightDetailPresenter.java
package com.porkandlager.travelonbudget.mvp.FlightDetail;

import android.content.Intent;

/**
 * Created by tista on 11/15/16.
 */

interface FlightDetailPresenter {

    void parseIntent(Intent intent);
    void bookNowClicked();

}

File: app/src/main/java/com/porkandlager/travelonbudget/mvp/FlightDetail/FlightDetailPresenter.kt
package com.porkandlager.travelonbudget.mvp.FlightDetail

import android.content.Intent

import com.porkandlager.travelonbudget.wires.models.responses.FlightDetailImageSearchResponse

/**
 * Created by tista on 11/15/16.
 */

internal interface FlightDetailPresenter {

    fun parseIntent(intent: Intent)
    fun bookNowClicked()
    fun showPhotosFromImageSearch(response: FlightDetailImageSearchResponse)

}
